ajax/libs/readremaining.js/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "readremaining.js",
+  "version": "1.0.0",
+  "filename": "src/output.min.js",
+  "description": "A jQuery plugin that allows you to show the user, how long will take to read the article taking in consideration the speed in which he is scrolling.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Aerolab/readremaining.js"
+  },
+  "keywords": [
+    "jquery",
+    "ui",
+    "read",
+    "timing"
+  ],
+  "author": "Aerolab",
+  "license": "MIT",
+  "homepage": "https://aerolab.github.io/readremaining.js/",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/Aerolab/readremaining.js.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "+(css|src)/*"
```

Comment:
You seemed to want to shorten the length of `files`, but should `src` be in `basePath`?